docs/hpc/about/pricing-compute.md

```diff
@@ -1,33 +1,53 @@
 # Pricing for Extra Compute
 
-_Example specifications and pricing as of February 2021_
+_Example specifications and pricing as of May 2023_
 
+Please note: You cannot be charged for the hardware until it has been received and invoiced. Since lead times can be extensive, please plan as far ahead as you can. 
 
 ## Pricing for prioritized compute ("CPU")
 
-Cluster compute nodes are purchased by Wynton in units which contain 4 Compute Nodes in a single chassis. Partial nodes can be purchased and contributions will be pooled towards a full system.
+Cluster compute nodes are purchased by Wynton in units that have 4 Compute Nodes in a single chassis. If partial nodes are requested, contributions will be pooled with other requests towards a full system. Another option is to pay $170/per member.q slot/share. 
 
-* CPU/node: 2 × Intel Xeon Gold 6336Y 2.4 GHz 24-core processors
-* RAM/node: 256 GiB
+**Price per 4 compute node unit: ~$33k USD excluding tax (for 4 nodes in a 2U chassis)**
 
-**Price per 4 compute node unit: 33,405 USD (for 4 nodes in a 2U chassis)**
+SMC RM224Q 2U Quad-Node Server 
+Four Hot-Pluggable Compute Modules, each Node Includes:  
+CPU: (2) Xeon Gold 6336Y 2.4GHz Twenty-Four-core  
```

Comment:
I can do this, but if you want another "exercise", we should clarify what `(2)` and `(4)` means by updating:

```
* CPU: (2) Xeon Gold 6336Y 2.4GHz Twenty-Four-core
```

to how we did it previously, i.e.

```
* CPU: 2 × Xeon Gold 6336Y 2.4GHz Twenty-Four-core
```

I would also spell out that it's an Intel processor and replace the wordy core specification as we did before, i.e.

```
* CPU: 2 × Intel Xeon Gold 6336Y 2.4GHz 32-core processor
```